hack/generator/pkg/codegen/pipeline_swagger_augment_status.go

```diff
@@ -17,96 +17,35 @@ import (
 	"sync"
 
 	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog/v2"
```

Comment:
This file is technically not a pipeline stage anymore -- wondering if it should be renamed and/or moved someplace else? (A subdirectory of codegen, or maybe into astmodel?)